api/v1alpha1/flowcollector_webhook.go

```diff
@@ -162,3 +165,11 @@ func Convert_v1beta1_FlowCollectorExporter_To_v1alpha1_FlowCollectorExporter(in
 func Convert_v1beta1_FlowCollectorEBPF_To_v1alpha1_FlowCollectorEBPF(in *v1beta1.FlowCollectorEBPF, out *FlowCollectorEBPF, s apiconversion.Scope) error {
 	return autoConvert_v1beta1_FlowCollectorEBPF_To_v1alpha1_FlowCollectorEBPF(in, out, s)
 }
+
+// // This function need to be manually created because conversion-gen not able to create it intentionally because
+// // we have new defined fields in v1beta1 not in v1alpha1
+// // nolint:golint,stylecheck,revive
+// func Convert_v1beta1_CertificateReference_To_v1alpha1_CertificateReference(in *v1beta1.CertificateReference, out *CertificateReference, s apiconversion.Scope) error {
```

Comment:
nit: looks like some garbage commented out?

Reply:
Yes, removed, thank you!